### DISCUSSION TITLE
Loading in datasets from private S3 bucket

### DISCUSSION THREAD

**Discussion by zejian99** [azavea/raster-vision#1894] [Q&A]
Hey, may I check how can I load in datasets into rastervision via RasterioSource if my data is stored in a private s3 bucket?

**Comment by AdeelH** [comment]
It should work automatically if your AWS credentials and already configured and stored in `~/.aws/`. If your AWS account requires an SSO login, you will need to run `aws sso login` first. LMK if you encounter any problems.
